Sources/SwiftCurrent_SwiftUI/Extensions/SwiftUIFlowPersistenceAdditions.swift

```diff
@@ -0,0 +1,46 @@
+//  swiftlint:disable:this file_name
+//  SwiftUIFlowPersistenceAdditions.swift
+//  SwiftCurrent_SwiftUI
+//
+//  Created by Morgan Zellers on 9/13/21.
+//  Copyright © 2021 WWT and Tyler Thompson. All rights reserved.
+//
+import SwiftCurrent
+
+extension FlowPersistence {
+    /// A namespace for the SwiftUI persistence types.
+    public enum SwiftUI {
+        /// A type indicating how a `FlowRepresentable` should be persisted.
+        public enum Persistence: RawRepresentable, CaseIterable {
```

Comment:
CaseIterable conformance is not strictly necessary, we do not use it for our tests. It also doesn't necessarily hurt anything, but I just thought I'd call out that it was superfluous

Reply:
removed